db/background_mgr.go

```diff
@@ -124,9 +124,12 @@ func (b *BackgroundManager) Start(ctx context.Context, options map[string]interf
 		return err
 	}
 
+	// disassociate with request context lifespan for background goroutines
+	backgroundCtx := context.WithoutCancel(ctx)
```

Comment:
I don't know if this is correct, at least not for a generic background task. The context used by the db init manager task, so is the concern that you could still use a background manager even after you stop the database?

My concern about having it be non cancellable is you might actually expect to be able to use this for cancellation, and generally having any work on the background manager will be non functional. It is weird to have a terminator and a context, but I'm just not sure why this is the way it is. Normally you'd need a bucket for the lifetime of a background job.

Reply:
I think this was an overzealous bugfix. Isn't required here which would affect other things more generally than the bug I was trying to fix. Reverted.